Hotfix needed for new binutils (bug is in gcc)
(see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=16625)

--- ./libkdegames/kgrid2d.h.orig	2004-08-13 13:52:56.000000000 +0200
+++ ./libkdegames/kgrid2d.h	2004-08-13 13:54:02.000000000 +0200
@@ -291,10 +291,14 @@
         case Right:     return c + Coord( 1,  0);
         case Up:        return c + Coord( 0, -1);
         case Down:      return c + Coord( 0,  1);
+	}
+        switch (n) {
         case LeftUp:    return c + Coord(-1, -1);
         case LeftDown:  return c + Coord(-1,  1);
         case RightUp:   return c + Coord( 1, -1);
         case RightDown: return c + Coord( 1,  1);
+	}
+        switch (n) {
         case Nb_Neighbour: Q_ASSERT(false);
         }
         return c;
